server/app/test/all_get/test_get_present.py

```diff
@@ -0,0 +1,16 @@
+#Get all orders N.B When the list is populated
+def test_get_all(create_test_app):
+    '''Test for get call orders'''
+    response = create_test_app.get('/v1/orders/present')
+    assert response.status_code == 200
+    assert b"No Items present" not in response.data
```

Comment:
rather than reading bytes, read on how you can jsonify your response, can be done later though

Reply:
Will look into that in due time